robot.js

```diff
@@ -82,7 +82,13 @@
       table.appendChild(tr);
       let rowHtml = '';
       for (let col = 0; col < cells.length; col++) {
-        const cell = cells[col] === '.' ? '' : cells[col];
+        // const cell = cells[col] === '.' ? '' : cells[col];
```

Comment:
You can take out this commented-out code. Consider a pull request as a final version that you want the reviewer to review. In general, a reviewer is not interested in 'old' code that used to be.